hal/devices/sierra_wireless/wp7702/impl.c

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2019-2020 BiiLabs Co., Ltd. and Contributors
```

Comment:
Let's move `devices` from `hal`. That is, we have two directories:
* `hal`: the abstract interface which allows high-level access;
* `devices`: the real device manipulations;